### PR TITLE
[RFE] Add openscap functionality to appliance_console_cli

### PR DESCRIPTION
**Pull request by rh-dluong**
Added functionality to appliance_console_cli for openscap
Enabled by the --openscap flag

Resolves https://bugzilla.redhat.com/show_bug.cgi?id=1759859

### CODE REVIEW

File: lib/manageiq/appliance_console/cli.rb [hunk 1]
```diff
@@ -351,6 +361,11 @@ def uninstall_ipa
       config.deactivate if config.ipa_client_configured?
     end
 
+    def scap
```
Comment (gtanzillo) [inline lib/manageiq/appliance_console/cli.rb]: Should the be `def openscap`?

File: lib/manageiq/appliance_console/cli.rb [hunk 2]
```diff
@@ -146,6 +153,7 @@ def parse(args)
         opt :http_cert,            "install certs for http server",     :type => :boolean
         opt :extauth_opts,         "External Authentication Options",   :type => :string
         opt :server,               "{start|stop|restart} actions on evmserverd Server",   :type => :string
+        opt :openscap, "Setup OpenScap", :type => :boolean, :default => false
```
[the remark this answers is not in the document]
Comment (rh-dluong) [reply lib/manageiq/appliance_console/cli.rb]: I think I tried that in my last commit and rubocop yelled at me. Haha.

File: lib/manageiq/appliance_console/cli.rb (hunk 1 above)
Comment (rh-dluong) [reply lib/manageiq/appliance_console/cli.rb]: Whoops, yep, must've forgot to change this when I was copying it over from my test appliance.

**Comment by rh-dluong** [comment]
@gtanzillo Rebased since it was unmergable at one point, but codeclimate is still complaining about that section with all the `||`

**Comment by Fryguy** [comment]
@carbonin Can you review this one?
@rh-dluong Can you rebase?

**Pull request merged**